app/workers/generate_goods_nomenclatures_report_worker.rb

```diff
@@ -0,0 +1,29 @@
+class GenerateGoodsNomenclaturesReportWorker
```

Comment:
report is a bit generic
what about
`GenerateGoodsNomenclaturesCsvWorker` ?
it's even easier to search

Reply:
Done